app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,9 @@
   </head>
 
   <body>
+    <% flash.each do |message_type, message| %>
+      <div class="alert alert-<%= message_type %>"><%= message %></div>
```

Comment:
remove/change the class? I think this one is my bad cuz I copy pasta'd from a previous project using bootstraps.